v22.1/security-reference/transport-layer-security.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Transport Layer Security (TLS) and Public Key Infrastructure (PKI)
+summary: Overview of PKI and TLS and how to implement them with CockroachDB
+toc: true
+docs_area: reference.security
+---
+
+This page provides a conceptual overview of Transport Layer Security (TLS) and the related notion of Public Key Infrastructre (PKI), and sketches the security-architecture considerations in play when using CockroachDB.
+
+See:
+
+- [Using Google Cloud Platform to manage PKI certificates.](../manage-certs-gcloud.html)
+- [Using the CockroachDB CLI to provision a development cluster.](../manage-certs-cli.html)
+
+## What is Transport Layer Security (TLS)?
+
+How do you send data over a network connection to a specific receiving party, and guarantee that nobody else can eavesdrop on the message or tamper with the contents? For much modern network communication, the Transport Layer Security (TLS) protocol is part of the solution. This protocol is used to establish securely authenticated and encrypted traffic between a client (the party who initiates the session) and a server (the party receiving the connection request).
```

Comment:
As a general comment for if/when you're making changes, might want to avoid rhetorical questions. My feeling that they're totally suitable for technical product docs where we want to be clear and free from any opinions (that aren't recs). It feels more like a narrative device to lead a reader, which we don't want to do.

Reply:
It's not really a rhetorical question, it's the framing of a problem that is solved by TLS, and the first part of the article aims to describe how.
How about: 'Network communication poses a very general problem: "How do you send..." '